### DISCUSSION TITLE
Raspberry Pi 400

### DISCUSSION THREAD

**Discussion by rastislavcore** [catchthatrabbit/coreminer#26] [Profitability]
### Model

Raspberry Pi 400

### CPU

Broadcom BCM2711 quad-core Cortex-A72 (ARM v8) 64-bit SoC @ 1.8GHz

### RAM

4GB LPDDR4-3200

### Hashrate

360 h/s

**Comment by BA777** [comment]
Hi, Raspberry Pi 4 is out of stock everywhere. What other MC would you suggest? I'm new to mining, so if you have a minimum spec information to share with me as well as a few alternative MC options, that would be super helpful and appreciated!

**Comment by rasdin-dunkelwald** [comment]
Hello raisty, thanks for sharing. Which OS are you using on the Pi 400?

**Reply by rastislavcore** [reply]
Latest Kali Linux for ARM. Here are the choices: https://www.kali.org/get-kali/#kali-arm

**Reply by mmetzkes** [reply]
thank you very much for your answer

**Reply by rasdin-dunkelwald** [reply]
thank you Raisty
